src/main/java/io/pinecone/clients/Pinecone.java

```diff
@@ -14,26 +14,57 @@
 
 public class Pinecone {
 
-    private final ManageIndexesApi manageIndexesApi;
+    private static ManageIndexesApi manageIndexesApi;
```

Comment:
Reason for changing this to static?

Reply:
It's instantiated in `Client.Build` and the compiler was complaining.

Reply:
ah, I see. I'll make a code suggestion

Reply:
I think we do not want to make it static or else the same index instance will be shared across all instances of Pinecone, which is probably not what we want

Reply:
We definitely don't, thanks for the callout. I've updated to leverage the pattern you recommended of passing `ManageIndexesApi` to the constructor.